.github/workflows/e2e-rocketmq.yaml

```diff
@@ -0,0 +1,54 @@
+#  Copyright 2021 SkyAPM
```

Comment:
Merge this file into `.github/workflows/e2e-pulsar.yaml` and revert the file name

Reply:
Thanks for you review, according to suggestion from my mentor @dmsolr , I divided it to two files. The reasons are as follows.
> they are building provider and consumer images by self on one VM. if they are same name in compose file, they will > get failed. another, they are two unrelated tasks.